SharpSource/SharpSource.Test/ThrowNullTests.cs

```diff
@@ -12,13 +12,31 @@ public class ThrowNullTests : DiagnosticVerifier
     protected override DiagnosticAnalyzer DiagnosticAnalyzer => new ThrowNullAnalyzer();
 
     [TestMethod]
-    public async Task ThrowNull_ThrowsNull()
+    public async Task ThrowNull_ThrowsNullLiteral()
     {
         var original = @"throw null;";
 
         await VerifyDiagnostic(original, "Throwing null will always result in a runtime exception");
     }
 
+    [TestMethod]
+    public async Task ThrowNull_ThrowsNullConstant()
+    {
+        var original = @"
+const System.Exception NullConst = null;
+throw NullConst;";
```

Comment:
Here, the exception is not `IConversionOperation`. So previously this didn't report a diagnostic.